app/models.py

```diff
@@ -131,17 +131,27 @@ def current_user_is_driver(self):
     def driver(self):
         return Person.query.get(self.driver_id)
 
+    def get_riders_query(self, statuses):
```

Comment:
nit: by convention i believe this should be `get_riders` b/c it returns RIDERS rather than a sql alchemy query object (see: `get_ride_requests_query` returns a query object that you can then get stuff back from by calling `all()`, `one_or_none()`, etc.

Reply:
Good point.

Reply:
Also, not a nit.